### ISSUE TITLE
Feature: [searh filter] [date range] make default field view more intuitive

### ISSUE THREAD

**Issue by AleksandrSokolov** [saltcorn/saltcorn#519]
By default date range field show just empty string.

I think that more intuitive will be show text "From" to "To" as in example:

![image](https://user-images.githubusercontent.com/327030/104692987-f15af100-5719-11eb-9aed-28c6f0922c81.png)

![image](https://user-images.githubusercontent.com/327030/104693117-29623400-571a-11eb-9434-4d79d1979feb.png)

May be also will be more clear if data range will be splitted to 2 fields: From / To like in example from other site:
![image](https://user-images.githubusercontent.com/327030/104693194-4bf44d00-571a-11eb-8146-8dcc9e80f666.png)

**Comment by j2l** [comment]
I also agree that the empty date range filter to get the calendar view is obscure from an user PoV.
